elixir-graphql/san_graphql_ex/lib/san_graphql_ex.ex

```diff
@@ -0,0 +1,67 @@
+defmodule SanGraphqlEx do
+  require Logger
+
+  @doc ~s"""
+  Returns a list of maps with `activeAddresses` and `datetime` keys.
+  """
+  def daily_active_addresses() do
+    case daily_active_addresses_query do
+      {:ok, response} ->
+        %Neuron.Response{body: %{"data" => %{"dailyActiveAddresses" => active_addresses}}} =
+          response
+
+        active_addresses
+
+      error ->
+        Logger.error("Cannot fetch daily active addresses. Reason: #{inspect(error)}")
+    end
+  end
+
+  @doc ~s"""
+  Returns a list of all project identifiers. These project identifiers can be used to query
+  data for a specific project. The identifier should be provided to the `slug` field
+  as seen in the `daily_active_addresses/0` example.
+  """
+  def project_slugs() do
+    case project_slugs_query() do
+      {:ok, response} ->
+        %Neuron.Response{body: %{"data" => %{"allProjects" => all_projects}}} = response
+
+        all_projects
+        |> Enum.map(fn %{"coinmarketcapId" => cmc_id} -> cmc_id end)
+
+      error ->
+        Logger.error("Cannot fetch projects' slugs. Reason: #{inspect(error)}")
+    end
+  end
+
+  # Private functions
+
+  defp project_slugs_query() do
+    Neuron.Config.set(url: "https://api.santiment.net/graphql")
+
+    Neuron.query("""
+    {
+      allProjects{
+        coinmarketcapId
```

Comment:
we should get `slug` instead of `coinmarketcapId` - its better for the public use

Reply:
We have no option - to use `slug` we need to change the GQL schema in sanbase.

Reply:
it's available: https://api.santiment.net/graphiql?query=%7B%0A%20%20allProjects%7B%0A%20%20%20%20slug%0A%20%20%7D%0A%7D